scripts/build_release.sh

```diff
@@ -95,6 +95,11 @@ make build
 export BUILD_DEB=1
 scripts/build_packages.sh "${PLATFORM}"
 
+# build docker release package
+pwd
+cd ../docker/release
+./build_algod_docker.sh ${HOME}/node_pkg/node_dev_linux-amd64_${FULLVERSION}.tar.gz
+cd ../../scripts
```

Comment:
Once you merge, replace this with `cd ${REPO_ROOT}/scripts`

Reply:
Merged and updated.